nixos/modules/services/networking/shout.nix

```diff
@@ -6,6 +6,21 @@ let
   cfg = config.services.shout;
   shoutHome = "/var/lib/shout";
 
+  defaultConfig = pkgs.runCommand "config.js" {} ''
+    EDITOR=true ${pkgs.shout}/bin/shout config --home $PWD
+    mv config.js $out
+  '';
+
+  configFile = if cfg.configFile then cfg.configFile else ''
```

Comment:
Won't this produce a type error? At least in my repl, if does not cast things to bool. Perhaps use `if  cfg.configFile != null` instead.